test/distributed/org/apache/cassandra/distributed/upgrade/MixedModeUnloggedBatchTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.distributed.upgrade;
+
+import org.junit.Test;
+
+import org.apache.cassandra.distributed.shared.Versions;
+
+public class MixedModeUnloggedBatchTest extends MixedModeBatchTestBase
+{
+    @Test
+    public void testSimpleStrategy22to30() throws Throwable
```

Comment:
It seems that this is missing 3.0 to 3.x, while we test that path for logged batches, is that intentional? Also, the tests for logged batches are split into two separate classes, while the tests for unlogged batches are inside a single class.

Reply:
Just an oversight. Will fix...